sdk/webpubsub/Microsoft.Azure.WebPubSub.Common/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Release History
 
+## 1.4.0 (2024-09-10)
+- The returned type of `MqttConnectEventRequest.ConnectionContext` and `MqttDisconnectedEventRequest.ConnectionContext` has been changed to `MqttConnectionContext`.
```

Comment:
sounds like a breaking change? what's the impact to customer?

Reply:
Yes, it's a breaking change... Now it's reverted.